### ISSUE TITLE
OCaml snippets outdated

### ISSUE THREAD

**Issue by ComanderP** [hmemcpy/milewski-ctfp-pdf#345]
Hello,

First off, I’ve been really enjoying the OCaml version of the book. It’s been a great read so far, and I appreciate the effort that’s gone into adapting the content for different programming languages.

However, while reading, I’ve noticed that some of the OCaml code snippets seem to be outdated for more recent versions of the language (e.g. usage of libraries that are discouraged in modern OCaml, and comments about/definitions of functions that are now in the standard library). There's also some snippets that seem to be misplaced.

I’d be happy to help update the code snippets to match a more recent version of OCaml. If someone could guide me on what I need to touch to change the snippets, I would appreciate it! I assume it's the code inside the chapters in `src/content/`, but I'd like to be prepared.

Also, again, thanks for creating such an awesome resource.

**Comment by drupol** [comment]
Yes that's pretty much it. Modify something then to build a local version of the book, I recommend using Nix, as described in the Readme.

Ping us in case of help.
